bundles/AdminBundle/Resources/public/js/pimcore/document/tags/areablock.js

```diff
@@ -22,6 +22,7 @@ pimcore.document.tags.areablock = Class.create(pimcore.document.tag, {
         this.id = id;
         this.name = name;
         this.elements = [];
+        this.elementCounter = [];
```

Comment:
Should we name this `brickTypeUsageCounter` instead? Or something else which is a bit more specific and self describing?

Reply:
changed it to `brickTypeUsageCounter`. thanks!